src/main/java/de/tum/in/www1/artemis/repository/UserRepository.java

```diff
@@ -65,7 +65,7 @@
     Page<User> findAllWithGroups(Pageable pageable);
 
     @Modifying
-    @Query("Update User user set user.lastNotificationRead = utc_timestamp where user.id = :#{#userId}")
+    @Query("Update User user set user.lastNotificationRead = current_timestamp() where user.id = :#{#userId}")
```

Comment:
@sascha11110 You have more in-depth knowledge about the notifications and timestamps than I do. I believe this should be `current_timestamp` instead of `utc_timestamp`? With the latter, I always ended up getting a wrong ZonedDateTime read from the db (zoned to Berlin with +01:00 but time was UTC).

Reply:
I agree this should be current_timestamp